### PR TITLE
[WIP] Updates to the create from URL section

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Updates:
- Added more information to make it clear
- Updated examples to include the hostname and html button code

To be done:
- Fix examples

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
[Update the Create from URL experience based on the examples provided by Eric Rich](https://trello.com/c/MitOw6hb/538-update-the-create-from-url-experience-based-on-the-examples-provided-by-eric-rich)

**Comment by gaurav-nelson** [comment]
@sferich888 Can you provide examples for 'Using an Image Stream and Image Tag' and 'Using a Template' section, which I can use here.

File: dev_guide/create_from_url.adoc [hunk 1]
```diff
@@ -24,13 +23,19 @@ To add namespaces to the whitelist, see
...
 image::run_on_openshift_buttons.png[Run on OpenShift Buttons]
 
 These buttons leverage a defined URL pattern with an appropriate query string.
-The user is prompted to select the project. Then, the Create from URL workflow
-continues.
+The user is prompted to select the project. Then, the Create from URL workflow continues.
+
+Each valid URL consists for two parts:
+
+* `Hostname` - Specifies where to deploy the application. This is the link to the xref:../architecture/infrastructure_components/web_console.adoc[Web Console] of an OpenShift Instance on which you want to deploy your application. For example, to create a URL which deploys your application on OpenShift Online, use `pass:[https://openshift.redhat.com/app/console/]` as `hostname`.
```
Comment (gaurav-nelson) [inline dev_guide/create_from_url.adoc]: @sferich888 is this example okay?

**Comment by kalexand-rh** [comment]
@gaurav-nelson, should we keep this PR open?

**Comment by gaurav-nelson** [comment]
Thanks @kalexand-rh we can close this, I will track this in Trello. Thanks.

**Pull request closed without merging**